python/pyspark/sql/column.py

```diff
@@ -151,6 +162,8 @@ def __init__(self, jc):
     __rdiv__ = _reverse_op("divide")
     __rtruediv__ = _reverse_op("divide")
     __rmod__ = _reverse_op("mod")
+    __pow__ = _bin_func_op("pow")
```

Comment:
@davies, `pow` function in `pyspark.sql.functions` is created with function `_create_binary_mathfunction` which uses `Column` internally, thus it cannot be simply imported from `pyspark.sql.fuction`